### PR TITLE
fix(kre-py): Logging issues

### PR DESCRIPTION
**Pull request by blopezpi**
Hi Team,
This is a fix to see the logs from the kre ui.

### CODE REVIEW

File: kre-py/Dockerfile [hunk 1]
```diff
@@ -62,4 +62,4 @@ WORKDIR /kre
 
 COPY --chown=kre:0 src /kre/src
 
-CMD ["/usr/bin/python3.9", "/kre/src/main.py", "2>&1 | tee -a /var/log/app/app.log"]
+CMD ["sh","-c","/usr/bin/python3.9 /kre/src/main.py 2>&1 | tee -a /var/log/app/app.log"]
```
Comment (aacecandev) [inline kre-py/Dockerfile:65]: L41 already makes python3.9 the default python version, why not using `["python", ...] directly?`

**Review by aacecandev — APPROVED** [review]

File: kre-py/Dockerfile (hunk 1 above)
Comment (blopezpi) [reply kre-py/Dockerfile:65]: Because, it doesn't work.

Comment (blopezpi) [reply kre-py/Dockerfile:65]: I will try to use the PYTHONPATH env.

**Comment by aacecandev** [comment]
Also, I would use either `PIPENV_DOTENV_LOCATION` or `PIPENV_VENV_IN_PROJECT` in order to have strict control of where the virtual environment is located

**Comment by blopezpi** [comment]
> Also, I would use either `PIPENV_DOTENV_LOCATION` or `PIPENV_VENV_IN_PROJECT` in order to have strict control of where the virtual environment is located

It's not possible to use a virtualenv. This is a base runner.

File: kre-py/Dockerfile (hunk 1 above)
Comment (blopezpi) [reply kre-py/Dockerfile:65]: It doesn't work :(

**Pull request merged**